scripts/screens/ClanScreen.py

```diff
@@ -343,36 +345,33 @@ def choose_cat_positions(self):
                 continue
 
             # Newborns are not meant to be placed. They are hiding. 
-            if Cat.all_cats[x].status == 'newborn' or game.config['fun']['all_cats_are_newborn']:
+            if Cat.all_cats[x].status.is_newborn() or game.config['fun']['all_cats_are_newborn']:
                 if game.config['fun']['all_cats_are_newborn'] or game.config['fun']['newborns_can_roam']:
                     # Free them
                     Cat.all_cats[x].placement = self.choose_nonoverlapping_positions(first_choices, all_dens,
                                                                                      [1, 100, 1, 1, 1, 100, 50])
                 else:
                     continue
 
-            if Cat.all_cats[x].status in ['apprentice', 'mediator apprentice']:
+            if Cat.all_cats[x].status.is_app_any():
```

Comment:
`is_app_any` includes medicine cat apprentice. Not equivalent.

Reply:
Good catch, thank you. This was before we had `can_patrol_app` so I'll replace it with that.